workflows/taxprofiler.nf

```diff
@@ -111,16 +111,16 @@ workflow TAXPROFILER {
         SUBWORKFLOW: PERFORM PREPROCESSING
     */
     if ( params.perform_shortread_clipmerge ) {
-
         ch_shortreads_preprocessed = SHORTREAD_PREPROCESSING ( INPUT_CHECK.out.fastq ).reads
+        ch_versions = ch_versions.mix( SHORTREAD_PREPROCESSING.out.versions )
     } else {
         ch_shortreads_preprocessed = INPUT_CHECK.out.fastq
     }
 
     if ( params.perform_longread_clip ) {
         ch_longreads_preprocessed = LONGREAD_PREPROCESSING ( INPUT_CHECK.out.nanopore ).reads
                                         .map { it -> [ it[0], [it[1]] ] }
-        ch_versions = ch_versions.mix(LONGREAD_PREPROCESSING.out.versions.first())
+        ch_versions = ch_versions.mix( LONGREAD_PREPROCESSING.out.versions )
```

Comment:
Isn't `first()` operator necessary?

Reply:
In this case no, as it's coming out of a subworkflow - so theoretically we may be getting multiple versions from all the modules

We only need to use `.firat()` when it's coming out of a module.